src/Microsoft.AspNetCore.DataProtection.Redis/Microsoft.AspNetCore.DataProtection.Redis.csproj

```diff
@@ -4,6 +4,7 @@
     <Description>Redis storage support as key store.</Description>
     <VersionPrefix Condition="'$(ExperimentalProjectVersionPrefix)' != ''">$(ExperimentalProjectVersionPrefix)</VersionPrefix>
```

Comment:
We also need to flow through ExperimentalVersionSuffix since this may vary from VersionSuffix

Reply:
`ExperimentalVersionSuffix` or `ExperimentalProjectVersionSuffix`?